cg/apps/mip/confighandler.py

```diff
@@ -0,0 +1,122 @@
+from copy import deepcopy
+import logging
+from pathlib import Path
+
+from marshmallow import Schema, fields, validate
+import ruamel.yaml
+
+from cg.exc import ConfigError
+from cg.constants import DEFAULT_CAPTURE_KIT, NO_PARENT
+
+
+LOG = logging.getLogger(__name__)
+
+
+class SampleSchema(Schema):
+    sample_id = fields.Str(required=True)
+    analysis_type = fields.Str(
+        required=True,
+        validate=validate.OneOf(
+            choices=[
+                "tga",
+                "wes",
+                "wgs",
+            ]
+        ),
+    )
+    father = fields.Str(default="0")
+    mother = fields.Str(default="0")
+    phenotype = fields.Str(
+        required=True,
+        validate=validate.OneOf(choices=["affected", "unaffected", "unknown"]),
+    )
+    sex = fields.Str(required=True, validate=validate.OneOf(choices=["female", "male", "unknown"]))
+    expected_coverage = fields.Float()
+    capture_kit = fields.Str(default=DEFAULT_CAPTURE_KIT)
+
+
+class ConfigSchema(Schema):
+    case = fields.Str(required=True)
+    default_gene_panels = fields.List(fields.Str(), required=True)
+    samples = fields.List(fields.Nested(SampleSchema), required=True)
+
+
+class SampleSchemaRNA(Schema):
+    sample_id = fields.Str(required=True)
+    analysis_type = fields.Str(required=True, validate=validate.OneOf(choices=["wts"]))
+    father = fields.Str(default=NO_PARENT)
+    mother = fields.Str(default=NO_PARENT)
+    phenotype = fields.Str(
+        required=True,
+        validate=validate.OneOf(choices=["affected", "unaffected", "unknown"]),
+    )
+    sex = fields.Str(required=True, validate=validate.OneOf(choices=["female", "male", "unknown"]))
+    expected_coverage = fields.Float()
+    capture_kit = fields.Str(default=DEFAULT_CAPTURE_KIT)
+
+
+class ConfigSchemaRNA(Schema):
+    case = fields.Str(required=True)
+    default_gene_panels = fields.List(fields.Str(), required=True)
+    samples = fields.List(fields.Nested(SampleSchemaRNA), required=True)
+
+
+class ConfigHandler:
+    def make_pedigree_config(self, data: dict, pipeline: str = None) -> dict:
+        """Make a MIP pedigree config"""
+        self.validate_config(data=data, pipeline=pipeline)
+        config_data = self.parse_pedigree_config(data)
+        return config_data
+
+    @staticmethod
+    def validate_config(data: dict, pipeline: str = None) -> dict:
+        """Validate MIP pedigree config format"""
+        errors = {}
+        if pipeline == "mip-rna":
+            errors = ConfigSchemaRNA().validate(data)
+        else:
+            errors = ConfigSchema().validate(data)
+        if errors:
```

Comment:
```suggestion
        if not errors:
            return {}
```
Then move all logic below four spaces to left